run-vmtest/run-bpf-selftests.sh

```diff
@@ -1,40 +1,26 @@
 #!/bin/bash
 
-# run_selftest.sh will run the tests within /${PROJECT_NAME}/selftests/bpf
-# If no specific test names are given, all test will be ran, otherwise, it will
-# run the test passed as parameters.
-# There is 2 ways to pass test names.
+# This script runs the tests within ${GITHUB_WORKSPACE}/selftests/bpf
+# If no specific test names are given, all test runners will be
+# executed. Otherwise, executed runners passed as parameters.
+# There are 2 ways to pass test names.
 # 1) command-line arguments to this script
-# 2) a comma-separated list of test names passed as `run_tests` boot parameters.
-# test names passed as any of those methods will be ran.
+# 2) a comma-separated list of test names passed as `run_tests` boot
+#    parameters.
 
```

Comment:
this is probably more for historical reasons these days. Maybe we should support only one way. I believe the boot parameters approach is not used anymore since the move to vmtest. It was needed with the old image approach.

Reply:
Yeah. It looks like boot parameters isn't used anymore. I'll clean this up.